packages/metro/src/bundle/buildBundle.ts

```diff
@@ -40,7 +40,7 @@ export interface AssetData {
   files: string[];
 }
 
-async function buildBundle(
+export async function buildBundle(
```

Comment:
Any particular reason for removing `export default` in favour of named export?

Reply:
I can see this being the case in almost every file. Is this some kind of a convention? In CLI, we tend to create a single file for a function and prefer `export default`s